sections/header.liquid

```diff
@@ -164,7 +164,7 @@
       assign localization_forms = true
     endif
   -%}
-  <header class="header header--{{ section.settings.logo_position }} header--mobile-{{ section.settings.mobile_logo_position }} page-width{% if section.settings.menu_type_desktop == 'drawer' %} drawer-menu{% endif %}{% if section.settings.menu != blank %} header--has-menu{% endif %}{% if has_app_block %} header--has-app{% endif %}{% if social_links %} header--has-social{% endif %}{% if shop.customer_accounts_enabled %} header--has-account{% endif %}{% if localization_forms %} header--has-localizations{% endif %}">
+  <header class="header color-{{ section.settings.color_scheme }} header--{{ section.settings.logo_position }} header--mobile-{{ section.settings.mobile_logo_position }} page-width{% if section.settings.menu_type_desktop == 'drawer' %} drawer-menu{% endif %}{% if section.settings.menu != blank %} header--has-menu{% endif %}{% if has_app_block %} header--has-app{% endif %}{% if social_links %} header--has-social{% endif %}{% if shop.customer_accounts_enabled %} header--has-account{% endif %}{% if localization_forms %} header--has-localizations{% endif %}">
```

Comment:
I actually think this is breaking the gradient functionality for the header. The color schemes still apply but will only display the fallback solid color. 

We already have a `color-` and `gradient` class the the parent element on line 154 and I think having it here too is effectively overriding the gradient class behavior. I suspect if we just remove this instance of the class all will work as expected.

Reply:
I removed the code from line 167 and kept the one in line 154 and the color scheme is still working for both header and menu